apps/neuvector/neuvector/aat/00.yaml

```diff
@@ -77,4 +77,6 @@ spec:
           enabled: true
           annotations:
             traefik.ingress.kubernetes.io/router.tls: "true"
+            traefik.ingress.kubernetes.io/frontend-entry-points: https
```

Comment:
Is it worth including `http` as well in `frontend-entry-points`?

Reply:
have added